include/orc/orc.hpp

```diff
@@ -63,6 +63,8 @@ void register_dies(dies die_vector);
 
 std::string to_json(const std::vector<odrv_report>&);
 
+std::string version_json();
```

Comment:
I put this routine inside `orc.cpp` to encapsulate the usage of `nlohmann::json` to just that file.